tests/tck/features/aggregate/Agg.feature

```diff
@@ -418,6 +430,17 @@ Feature: Basic Aggregate and GroupBy
       | 45  | 13    |
       | 46  | 2     |
       | 47  | 3     |
+    When executing query:
```

Comment:
Do you have a case about nested agg call.

Reply:
Same as https://github.com/vesoft-inc/nebula-graph/pull/968#discussion_r616303222 ?